source/tools/testConsistency/ClientApp/views/index.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
Probably need the msft copy right header using the comment syntax here as well *<%# Comment tag, no execution, no output)

Reply:
Added